src/styles/blocks/header.scss

```diff
@@ -0,0 +1,59 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), padding and margin values should be written using multiples of 4. Consider changing the padding value to adhere to this rule, for example `padding: 0 48px;`.